Swiftfin/Views/ItemEditorView/ItemEditorView.swift

```diff
@@ -98,26 +101,28 @@ struct ItemEditorView: View {
     @ViewBuilder
     private var refreshButtonView: some View {
         Section {
-            RefreshMetadataButton(item: viewModel.item)
-        } footer: {
-            LearnMoreButton(L10n.metadata) {
-                LabeledContent(
-                    L10n.findMissing,
-                    value: L10n.findMissingDescription
-                )
-                LabeledContent(
-                    L10n.replaceMetadata,
-                    value: L10n.replaceMetadataDescription
-                )
-                LabeledContent(
-                    L10n.replaceImages,
-                    value: L10n.replaceImagesDescription
-                )
-                LabeledContent(
-                    L10n.replaceAll,
-                    value: L10n.replaceAllDescription
-                )
+            Button {
+                router.route(to: .itemMetadataRefresh(viewModel: metadataViewModel))
+            } label: {
+                HStack {
+                    Text(L10n.refreshMetadata)
+                        .foregroundStyle(.primary)
+
+                    Spacer()
```

Comment:
The `Text` should have `frame(maxWidth: .infinity, alignment: .trailing)` instead of a `Spacer`.